torch/testing/_core.py

```diff
@@ -275,6 +276,111 @@ def make_non_contiguous(tensor: torch.Tensor) -> torch.Tensor:
     return input.data
 
 
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, high=None,
+                requires_grad: bool = False, noncontiguous: bool = False,
+                exclude_zero: bool = False) -> torch.Tensor:
```

Comment:
For functions which have a lot of parameters, I suggest we use a more readable format at the cost of some vertical space:

```suggestion
def make_tensor(
    size: Tuple[int, ...],
    *,
    device: Union[str, torch.device] = "cpu",
    dtype: torch.dtype = torch.float32,
    low: Optional[float] = None,
    high: Optional[float] = None,
    requires_grad: bool = False,
    noncontiguous: bool = False,
    exclude_zero: bool = False
) -> torch.Tensor:
```

Reply:
Right, this makes sense. This should be fixed now.